components/Feature/AddAction/index.js

```diff
@@ -70,6 +70,8 @@ const AddAction = ({ hackneyToken, id, updatePlan }) => {
     });
     const plan = await response.json();
     if (plan) await updatePlan(plan);
+    location.reload(false);
```

Comment:
what does this do?

Reply:
it just reloads the page. It shouldn't have to be done, cause it should just rerender and display actions, I'll check if I can do it another way